packages/app/src/cli/services/dev/extension/utilities.ts

```diff
@@ -58,3 +58,14 @@ export function getExtensionPointTargetSurface(extensionPointTarget: string) {
       return domain
   }
 }
+
+function isCheckoutTargetsBeginsWithCustomerAccount(extensionPointTarget: string) {
+  const checkoutTargetsBeginsWithCustomerAccount = new Set([
+    'customer-account.order-status.cart-line-item.render-after',
+    'customer-account.order-status.cart-line-list.render-after',
+    'customer-account.order-status.customer-information.render-after',
+    'customer-account.order-status.block.render',
+  ])
+
+  return checkoutTargetsBeginsWithCustomerAccount.has(extensionPointTarget)
+}
```

Comment:
What should the DX look like when developing these extensions ? If a shop has no order, is it proper to send you to customer account although you need to go and complete a checkout first ?

Once all of those extension points are on ODP do you expect us to align the behaviour ?

Reply:
If a shop does not have an order, it may make sense to redirect to checkout page.
If a shop already has an order, it does not make any sense to redirect to checkout page. 

The issue is that cli does not know if the shop has an order or not. 

These are good questions. I will have a alignment with my team about this.